nbs/common.base_model.ipynb

```diff
@@ -46,7 +46,7 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #115.                
</pre><p>Please remove this extra line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>That line is still there.</p><p> </p>

Reply:
ah yes that should be gone now!